### ISSUE TITLE
When you use react-create-app along with uxi, you got errors

### ISSUE THREAD

**Issue by dervalp** [uxilab/uxi#53]
Step to reproduce:

- Install react-create-app and generate a dummy Application.
- install uxi
- try to use DataGrid

You get:

```
TypeError: _Flex2.default.extend is not a function
```

Some other components are also concerned.

It seems the way we are building webpack required some 'special' config, (which should not...)

**Comment by denisflorkin** [comment]
Yeah I'm guessign you install the latest styled-components (I just did the same) and it seems that extend is the only issue.
I also have other warns in v4 of styled-components (like for the .attrs(fn helpers)).

I can see two issue here:
- I add to manually install styled-compônents when using uxi (peerDep) maybe we should just make it a **direct deps** with a precise version number ? 

- NExt to that I'll set up a task in uxi to bring our compos to work **with styled-components v4**.
https://github.com/uxilab/uxi/issues/72
